js/ui/components/clubhouse.js

```diff
@@ -708,14 +624,18 @@ var Component = GObject.registerClass({
     }
 
     get _useClubhouse() {
-        return this._imageUsesClubhouse() && !!getClubhouseApp();
+        return this._imageUsesClubhouse() && !!this.getClubhouseApp();
+    }
+
+    getClubhouseApp() {
```

Comment:
I think this method is not necessary. There is already a getClubhouseApp function!

Reply:
This method is here to make the code generic to be used in the oldClubhouse class that inherits this one, so the component code doesn't know if it's the new or the old one and this.getClubhouseApp will return the component app. This can be replaced with `getClubhouseApp(this._clubhouseId)`, but this is a shortcut.